### ISSUE TITLE
closing new tickets doesn't generate email (Trac #1303)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1297]
When I close a "new" ticket via a commit, it does not generate email or go to the slack bot.

Migrated from https://code.icecube.wisc.edu/ticket/1303
```json
{
    "status": "closed", 
    "changetime": "2015-08-31T21:29:08", 
    "description": "When I close a \"new\" ticket via a commit, it does not generate email or go to the slack bot.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1441056548802590", 
    "component": "infrastructure", 
    "summary": "closing new tickets doesn't generate email", 
    "priority": "major", 
    "keywords": "", 
    "time": "2015-08-28T22:46:18", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150831T19:01:45: david.schultz commented:

> Depends on #986. Might even be fixed (or at least mitigated) by trac 1.1, so two tickets resolved by one upgrade.

**Comment by icecube-trac** [comment]
Trac update at 20150831T21:29:08: 
* nega changed resolution from "" to "fixed"
* nega changed status from "new" to "closed"
* nega commented:

> Looks like !ReadmeRender couldn't open its cache, and was crashing the plugin stack.
